package_drupal_script.sh

```diff
@@ -127,7 +127,7 @@ popd >/dev/null
 
 # clone or refresh d8 clone
 mkdir -p sprint
-git clone --quiet https://git.drupal.org/project/drupal.git ${STAGING_DIR}/sprint/drupal8
+git clone --config core.autocrlf=false --config core.eol=lf --quiet https://git.drupal.org/project/drupal.git ${STAGING_DIR}/sprint/drupal8
```

Comment:
This seems like a good opportunity for a couple of new test assertions?

```
cd ${SPRINTDIR}/${SPRINT_NAME}/drupal8
echo "# Test git configuration (${git config core.autocrlf})" >&3
AUTOCRLF=$(git config core.autocrlf)
[ "$(AUTOCRLF)" = "false" ]
```

Reply:
Great idea.